### PR TITLE
build: Remove external format checks

### PR DESCRIPTION
**Pull request by abhiomkar**
Removed external format checks to stay in sync with internal format.

### CODE REVIEW

**Review by asyncLiz — APPROVED** [review]
LGTM, but how come clang-format isn't in sync with internal TS formatting? Our config is just BasedOnStyle: Google. Do we need to update the dependency?

**Comment by abhiomkar** [comment]
TS Auto format is now enforced internally. I'll hold off merge this PR.

**Comment by dfreedm** [comment]
I do like the idea to at least unblock unit test runs from format checks. I'll do that in a separate PR

**Pull request merged**
